openlibrary/plugins/openlibrary/js/graphs/plot.js

```diff
@@ -79,10 +79,10 @@ export function loadEditionsGraph() {
                 y = item.datapoint[1].toFixed(0);
                 if (y == 1) {
                     showTooltip(item.pageX, item.pageY,
-                        `${y} $_('edition in') ${x}`);
+                        `${y} edition in ${x}`);
```

Comment:
Ripping out existing internationalization is moving the wrong direction. It looks like this got broken during the refactoring, so it should have been fixed then.

If we're going to keep this tooltip (I question it's value), it can be replaced with a simple universally understood notation like "N @YYYY" or just "N, YYYY" (assuming that the default tooltip isn't something similar already).

Reply:
The i18n was broken in a previous PR; it's currently broken. I'm not convinced those notations are universally understood; I think any speaker will under stand `1431 adfaeah af eifa si 2019` as number followed by year. I created an issue for adding a flow for client-side i18n https://github.com/internetarchive/openlibrary/issues/2607 to continue, so won't fix as part of this.

Reply:
Also there was a decent amount of discussion about this on the issue, so if you disagreed with the proposed solution, commenting there would have been more useful than commenting once the PR was open, after effort has been put into changing code. It's frustrating as a contributor to get agreement on the issue, open a PR, and get immediately shot down.